test/regression.uts

```diff
@@ -791,9 +791,11 @@ assert(len(tmp) == 1)
 = Sending an ICMP message 'forever' at layer 2 and layer 3
 ~ netaccess IP ICMP
 tmp = srloop(IP(dst="8.8.8.8")/ICMP(), count=1)
+tmp
```

Comment:
how is this helping? what is it supposed to trigger?

Reply:
Not really helpful. Removed